packages/arb-ts/scripts/deployStandard.ts

```diff
@@ -0,0 +1,131 @@
+import { instantiateBridge } from './instantiate_bridge'
+import { ERC20__factory } from '../src/lib/abi/factories/ERC20__factory'
+import dotenv from 'dotenv'
+import args from './getCLargs'
+import { constants, BigNumber, utils } from 'ethers'
+
+dotenv.config()
```

Comment:
why do we ask users to set `PRIVKEY` env variable in the readme if we're using dotenv?

Reply:
I don't follow - dotenv just let's you load env variables from .env if you want